From a33e70c36a968577a710eb73cd734d1aa87d41ae Mon Sep 17 00:00:00 2001
From: yanghuatao <yanghuatao@xiaomi.com>
Date: Mon, 4 Sep 2023 21:29:55 +0800
Subject: [PATCH 2/2] libm/libmcs: Fix clang build libmcs warning

Fix warnings:
libmcs/libmcs/libm/common/fenv.c libmcs/libmcs/libm/common/fenv.c:26:15: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
int fegetround()

Signed-off-by: yanghuatao <yanghuatao@xiaomi.com>
---
 libmcs/libmcs/libm/common/fenv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmcs/libmcs/libm/common/fenv.c libmcs/libmcs/libm/common/fenv.c
index df0fdc241..7cc50c876 100644
--- a/libmcs/libmcs/libm/common/fenv.c
+++ libmcs/libmcs/libm/common/fenv.c
@@ -23,7 +23,7 @@ int fesetexceptflag(const fexcept_t *flagp, int excepts)
     return -1;
 }
 
-int fegetround()
+int fegetround(void)
 {
     return -1;
 }
-- 
2.40.1

